gbalink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 include_directories(. ./../Runtime ${ATHENA_INCLUDE_DIR} ${JBUS_INCLUDE_DIR})
-add_executable(gbalink main.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl")
```

Comment:
ok, try switching back to master and pull the latest; update submodules too.. `gbalink` shouldn't have to link logvisor at all